doc/components.md

```diff
@@ -8,7 +8,7 @@ This diagram might look intimidating, but don't worry, it is not as complex as i
 All components inherit from the abstract class `Component`.
 
 If you want to skip reading about abstract classes you can jump directly to
-[PositionComponent](#positioncomponent).
+[](#positioncomponent).
```

Comment:
Is this intentional?

Reply:
Yes, it takes the title of the section and puts it there when it is rendered. :)

Reply:
Interesting!